daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -2643,6 +2644,15 @@ func NewDeviceSet(root string, doInit bool, options []string, uidMaps, gidMaps [
 		minFreeSpacePercent:   defaultMinFreeSpacePercent,
 	}
 
+	if driverVersion, err := devicemapper.GetDriverVersion(); err == nil {
```

Comment:
Actually, perhaps that function can be reused if we catch the error;
https://github.com/moby/moby/blob/bccebdac1806d39848c6b51e9fea757e77bc2dad/daemon/graphdriver/devmapper/deviceset.go#L1667-L1690

(should the driver status report if it's supported/not supported, so that it's visible in `docker info`? https://github.com/moby/moby/blob/bccebdac1806d39848c6b51e9fea757e77bc2dad/daemon/graphdriver/devmapper/deviceset.go#L2563-L2573